libs/logic-apps-shared/src/utils/src/lib/models/template.ts

```diff
@@ -8,9 +8,17 @@ export interface Manifest {
   skus: SkuType[];
   kinds?: WorkflowKindType[];
   details: Record<string, string>;
+
+  /* This is a markdown to show features for multi-workflow and details in case of single workflow */
   detailsDescription?: string;
+
   tags?: string[];
   artifacts: Artifact[];
```

Comment:
From our last discussion, we decided that artifacts will not be present for multi-workflow manifest. Should we consider making this an optional field?

Reply:
No, we can have artifacts at workflow app level, hence i decided to not make that change... lets see what we get once we do zipdeploy